src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -603,6 +604,16 @@ private function addTemplatingSection(ArrayNodeDefinition $rootNode)
         $rootNode
             ->children()
                 ->arrayNode('templating')
+                ->validate()
+                    ->ifTrue(function ($v) {
+                        return null !== $v['hinclude_default_template'];
+                    })
+                    ->then(function ($v) {
+                        @trigger_error('Using the key "hinclude_default_template" in the templating is deprecated since Symfony 4.3. Use the config key under fragments instead.', E_USER_DEPRECATED);
```

Comment:
any reason we don't use the setDeprecated() method instead?

Reply:
no reason, will change this